<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>768</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblMainTitle">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Aller Display&quot;;
color: rgb(255, 106, 0);
</string>
      </property>
      <property name="text">
       <string>Snejk!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="arena" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(189, 196, 255);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputPort"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Adres:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputAddress"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnConnect">
        <property name="text">
         <string>Połącz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Wyjdź</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTextEdit" name="txtLog">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
